docs/mobile/quick-start.mdx

```diff
@@ -40,57 +41,55 @@ No extra setup is required. You can install your emulator of choice, or debug on
+The quickstart is composed of four projects, one for each platform, and a common project that is shared by all platforms. This is a standard Xamarin Forms setup, as described [in their documentation](https://docs.microsoft.com/en-us/xamarin/get-started/quickstarts/deepdive?pivots=windows#anatomy-of-a-xamarinforms-application). Changes have been made to each project to support the bootstrapping of a Geocortex Mobile application using the SDK.
 
-<!-- ### Geocortex Mobile SDK Packages
+### Geocortex Mobile SDK Packages
 
-The Geocortex Mobile SDK is composed of four packages:
+The Geocortex Mobile SDK is composed of Geocortex packages:
 
--   `Geocortex API`
-    -   This package includes underlying code common to different Geocortex .NET Projects, include much of the code that interfaces with Esri's APIs.
+-   `Geocortex Icons`
+    -   This package provides configurable vector icons for use in Geocortex Mobile apps.
 -   `Geocortex Mobile`
     -   This package is the heart of the Mobile SDK, and powers the interface and interactions of a Mobile Xamarin Application.
+    -   This package includes the Geocortex.API namespace, an API for working with foundational Geocortex models, configuration, Esri's ArcGIS types, etc.
 -   `Geocortex Workflow`
     -   Geocortex Mobile heavily relies on [Geocortex Workflow](../workflow/overview.mdx) to power it's functionality.
+    -   Geocortex Workflow is itself composed of many packages (Image Activities, Forms, GIS, Runtime).
 -   `Geocortex Viewer Framework`
-    -   This package provides the infrastructure that powers the [layout](concepts-layout.mdx) and [app config](concepts-app-config.mdx) that is central to every Mobile App.
-
-TODO: Packages - wait on Kenny package publishing -->
+    -   This package provides the definitions of [layout](concepts-layout.mdx) and [app config](concepts-app-config.mdx) that are central to every Mobile app.
 
 ### Entry Points
 
-Each platform has platform specific setup and an entry point, after which they all boostrap the application through the platform agnostic project. In the platform agnostic project, the `OnStart` method of `App.cs` calls the Geocortex Mobile SDK Bootstrapper, and loads and displays the application with the given `app.json` configuration.
+Each platform has platform specific setup and an entry point, after which they all boostrap the application through the platform agnostic project. In the shared App1 project, the `OnStart` method of `App.cs` calls the Geocortex Mobile SDK Bootstrapper, and loads and displays the application with the given `app.json` configuration.
```

Comment:
I'm hesistant to used the "shared App1" name, because it doesn't hold up if someone renames their projects.

Reply:
Reverted to platform agnostic.